app/models/user.rb

```diff
@@ -2152,6 +2153,10 @@ def update_previous_visit(timestamp)
     update_column(:previous_visit_at, last_seen_at) if previous_visit_at_update_required?(timestamp)
   end
 
+  def change_display_name
+    Jobs.enqueue(:change_display_name, user_id: id, old_name: name_before_last_save, new_name: name)
```

Comment:
I think we need to consider what happens when a user changes their display name multiple times in a short period. Multiple jobs can be enqueued but sidekiq does not guarantee that it executes jobs based on the order it was enqueued.

Reply:
I think Sam just implemented limited concurrency per job class for the anonymization job. Maybe we can use the same strategy here.